src/less/components/payex/button.less

```diff
@@ -27,16 +30,30 @@
 	}
 
 	&.btn-secondary {
+		border: 1px solid var(--brand-secondary-light-3);
+
 		&:focus {
-			border: 2px solid @brand-primary;
+			border: 1px solid var(--brand-secondary-light-3);
 			box-shadow: none;
 		}
 
 		&:active {
-			border: 1px solid var(--brand-primary);
+			border: 3px solid @btn-active-border-color;
+		}  
+		
+		&:hover {
+			background-color: var(--light-gray) !important;
+			color: var(--brand-secondary) !important;
+			border: 1px solid var(--brand-secondary-light-3);
 		}
 	}
 
+ 	&.btn-tertiary {
+		&:hover {
+			background-color: var(--soft-brown) !important;
```

Comment:
do they all actually need the `!important` ?
If yes then go for it, but in general I'd really try to move away from it (since it's so much a "all or nothing" way to set specificity in CSS)